bigbluebutton-html5/.meteor/versions

```diff
@@ -4,62 +4,62 @@ allow-deny@1.0.5
 amplify@1.0.0
 arunoda:npm@0.2.6
 autoupdate@1.3.12
-babel-compiler@6.13.0
+babel-compiler@6.14.1
 babel-runtime@1.0.1
 base64@1.0.10
 binary-heap@1.0.10
-blaze@2.2.0
+blaze@2.3.0
 blaze-tools@1.0.10
 boilerplate-generator@1.0.11
 caching-compiler@1.1.9
-caching-html-compiler@1.0.7
+caching-html-compiler@1.1.0
 callback-hook@1.0.10
 cfs:http-methods@0.0.32
 cfs:micro-queue@0.0.6
 cfs:power-queue@0.9.11
 cfs:reactive-list@0.0.9
 cfs:reactive-property@0.0.4
-check@1.2.4
+check@1.2.5
 clinical:nightwatch@2.0.1
-coffeescript@1.11.1_4
+coffeescript@1.12.3_1
```

Comment:
I tried to clean that section, and even was able to delete some of the unused packages a few months ago.
But with these two - when you remove them, then Meteor restarts and you can see this:
![packages](https://cloud.githubusercontent.com/assets/9299997/23526589/9e3593aa-ff60-11e6-8d2c-515d6c89425f.JPG)